### PR TITLE
feat(patterns/container): use LMFR‘s container as default one

### PR DESCRIPTION
**Pull request by gael-boyenval**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a breaking change?

read [What is a breaking change ?](https://mozaic.adeo.cloud/Contributing/Prerequisite/GitConventions/#breaking-changes-)

- [ ] Yes
- [x] No

## Describe the changes

Github issue number or Jira issue URL: https://design-system-adeo.atlassian.net/browse/MZC-268

### CODE REVIEW

File: src/docs/Foundations/Layout/Grid/design.mdx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  scss: 'stable'
+---
+
+> These are the recommendations to set up the values to use as your layout settings in your Sketch files.
+
+# Layout behaviors
+
+Grids on breakpoints from `S` to `M` are fluid.
+
+![fluid-grid](fluid-grid.png)
```
Comment (gael-boyenval) [inline src/docs/Foundations/Layout/Grid/design.mdx]: The width and height properties does not work ? The one @nherwegh gave to you ? 
We need to use smaller size images

File: src/docs/Foundations/Layout/Container/index.mdx [hunk 2]
```diff
@@ -1,28 +1,32 @@
 ---
 title: 'Container'
-order: 2
+order: 3
 status:
-  sketch: 'wip'
-  scss: 'wip'
+  sketch: 'Bêta'
+  scss: 'Bêta'
 ---
```
Comment (gael-boyenval) [inline src/docs/Foundations/Layout/Container/index.mdx]: add a description, and can we add a thumbnail ?

File: src/docs/Foundations/Layout/Grid/index.mdx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title: 'Grid system'
-order: 3
+order: 2
 status:
   sketch: 'stable'
   scss: 'stable'
```
Comment (gael-boyenval) [inline src/docs/Foundations/Layout/Grid/index.mdx]: add a description, and can we add a thumbnail ?

File: src/docs/Foundations/Layout/Grid/design.mdx (hunk 1 above)
Comment (caqueste) [reply src/docs/Foundations/Layout/Grid/design.mdx]: I tried different variations of this code, It didn't work :/

File: src/docs/Foundations/Layout/Container/code.mdx [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ---
-title: 'Container'
+title: 'Code'
 order: 2
+description: 'The container define the main content width as well as the minimum negative space between the horizontal edges of the screen and the content.''
```
Comment (mohamedMok) [inline src/docs/Foundations/Layout/Container/code.mdx]: Additional single quote
`description: 'The container define the main content width as well as the minimum negative space between the horizontal edges of the screen and the content.'`

File: src/docs/Foundations/Layout/Container/index.mdx [hunk 5]
```diff
@@ -1,28 +1,33 @@
...
 
-> the container define the main content width as well as the minimum negative space between the horizontal edges of the screen and the content.
+> The container define the main content width as well as the minimum negative space between the horizontal edges of the screen and the content.
 
 ## Default container
 
-the default container follow those rules :
+The default container follows those rules :
```
Comment (nherwegh) [inline src/docs/Foundations/Layout/Container/index.mdx]: these not those

File: src/docs/Foundations/Layout/Grid/index.mdx [hunk 6]
```diff
@@ -1,29 +1,26 @@
...
+## Grids anatomy
 
-### Grids can be described using the following properties :
-
-- **column count** - the number of columns
-- **gutters** - the space between columns
-- **container width** - it can be either fixed (ex: 1200px) or fluid (take the available space)
-- **container paddings** - they create negative space between the edge of the grid (and therefore the content) and the edge of the container and need to be substracted from the container width to get the effective grid width.
+- **Columns**: It allows to place and structure the content.
+- **Gutters**: It's the space between the columns, it give air to the design and help to split the content.
```
Comment (nherwegh) [inline src/docs/Foundations/Layout/Grid/index.mdx]: …it gives the design some air and helps to split the content.

File: src/docs/Foundations/Layout/Grid/index.mdx [hunk 7]
```diff
@@ -1,29 +1,26 @@
...
+- **Margins**: It's the space between the edge of the screen and the container.
+- **Container**: It contains the columns and the gutters and define the content perimeter. It can be either fixed (ex: 1200px) or fluid (take the available space).
```
Comment (nherwegh) [inline src/docs/Foundations/Layout/Grid/index.mdx]: … and defineS…

File: src/docs/Foundations/Layout/Grid/index.mdx [hunk 8]
```diff
@@ -40,7 +37,7 @@ When using a grid, you decide the size of your elements by **the number of colum
 ![GridPossibilities](GridPossibilities.png)
 ![GridSpacing](GridSpacing.png)
 
-# The Leroy Merlin default Grid
+## The LEROY MERLIN default Grid
```
Comment (nherwegh) [inline src/docs/Foundations/Layout/Grid/index.mdx]: LEROY MERLIN's default grid

File: src/docs/Foundations/Layout/Container/code.mdx (hunk 4 above)
[the remark this answers is not in the document]
Comment (gael-boyenval) [reply src/docs/Foundations/Layout/Container/code.mdx]: A proposal : 

```
The container is the default main content wrapper. It set the maximum content's width and margins values for each supported viewports.
```

??

[the remark this answers is not in the document]
Comment (caqueste) [reply src/docs/Foundations/Layout/Container/code.mdx]: A proposal to be more simple:

A container is the available space between the left and right margins. It is composed by the columns and the gutters.

[the remark this answers is not in the document]
Comment (gael-boyenval) [reply src/docs/Foundations/Layout/Container/code.mdx]: Not true from a technical standpoint. You make it sound like it is a value, which it is not. And there is no references to responsive behaviors. Plus, a container is not necessarily composed of any columns.

[the remark this answers is not in the document]
Comment (nherwegh) [reply src/docs/Foundations/Layout/Container/code.mdx]: what about: 
The container defines the main content area with its external margins depending on the viewport's width.

**Pull request merged**